client/my-sites/comments/main.jsx

```diff
@@ -47,12 +52,33 @@ export class CommentsManagement extends Component {
 			translate,
 		} = this.props;
 
+		const jetpackUpdateLink = `/plugins/jetpack/${ this.props.siteSlug }`;
+
+		const showJetpackUpdateScreen =
+			isJetpack &&
+			! jetpackCommentsManagementSupported &&
+			config.isEnabled( 'comments/management/jetpack-5.5' );
+
 		return (
 			<Main className="comments" wideLayout>
 				<PageViewTracker path="/comments/:status/:site" title="Comments" />
 				<DocumentHead title={ translate( 'Comments' ) } />
-				<SidebarNavigation />
-				{ showPermissionError && (
+				{ showJetpackUpdateScreen && (
+					<EmptyContent
+						title={ preventWidows( translate( "Looking to manage this site's comments?" ) ) }
+						line={ preventWidows(
+							translate( 'We need you to update to the latest version of Jetpack' )
```

Comment:
Is this aggressive sounding? Let's add an editorial review label for their input.

Reply:
Added a "Needs Copy Review" Label for you 😄

Reply:
Maybe "Update to the latest version of Jetpack"?

Reply:
> Is this aggressive sounding? Let's add an editorial review label for their input.

Probably. :smile:  I just carried over the copy that was provided with designs in the issue.

Reply:
Updated copy in 5676b5222406d81fea2771233224456b58406758 to:
>Looking to manage this site's comments?
>Please update to the latest version of Jetpack first